snapshot/backup.go

```diff
@@ -485,41 +466,30 @@ func (snap *Builder) chunkifyDirpack(cIdx int, chk *chunkers.Chunker, pathname s
 			objectHasher.Write(chunkCopy)
 
 			if err := processChunk(i, chunkCopy); err != nil {
-				return nil, objects.MAC{}, -1, err
+				return objects.MAC{}, err
 			}
 		}
 		if err == io.EOF {
 			break
 		}
 	}
 
-	if totalDataSize > 0 {
-		object.Entropy = totalEntropy / float64(totalDataSize)
-	} else {
-		object.Entropy = 0.0
-	}
-
-	if object.ContentType == "" {
-		object.ContentType = mime.TypeByExtension(path.Ext(pathname))
-		if object.ContentType == "" {
-			object.ContentType = "application/octet-stream"
-		}
-	}
+	object.ContentType = "application/octet-stream"
 
 	copy(object_t32[:], objectHasher.Sum(nil))
 	object.ContentMAC = object_t32
 
 	objectSerialized, err := object.Serialize()
 	if err != nil {
-		return nil, objects.MAC{}, -1, err
+		return objects.MAC{}, err
 	}
 	objectMAC := snap.repository.ComputeMAC(objectSerialized)
 
-	if err := snap.repository.PutBlobWithHint(cIdx, resources.RT_OBJECT, objectMAC, objectSerialized); err != nil {
-		return nil, objects.MAC{}, -1, err
+	if err := snap.repository.PutBlobIfNotExistsWithHint(-1, resources.RT_OBJECT, objectMAC, objectSerialized); err != nil {
```

Comment:
add a comment to explain that we have to do this for dirpack (vs version without NotExists in regular chunkifier) because we already checked there.